src/py42/sdk/queries/query_filter.py

```diff
@@ -176,12 +180,33 @@ def __init__(self, term, operator, value=None):
         self._operator = operator
         self._value = value
 
+    @classmethod
+    def from_dict(cls, _dict):
+        return cls(_dict[u"term"], _dict[u"operator"], value=_dict.get(u"value"))
+
+    @property
+    def term(self):
```

Comment:
I exposed these so that they can be explored more easily when iterating through a list of filter groups and trying to do logic based on them